apps/src/signUpFlow/LoginTypeSelection.tsx

```diff
@@ -60,6 +60,24 @@ const LoginTypeSelection: React.FunctionComponent = () => {
     }
   }, [passwordIcon, confirmPasswordIcon, emailIcon]);
 
+  useEffect(() => {
```

Comment:
Can we not add a `onKeyDown` prop to the button because the design system doesn't support it? (I think I saw that comment from you somewhere but not in the PR description.)

Reply:
Correct- there's no onKeyDown prop the way our Legacy button supports it (or they way we write them in on <input components elsewhere).